src/screens/MainApp/ImageReview/index.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from 'react'
+import { View } from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome5'
+import { Button } from '~/components'
+import NavigationService from '~/navigationService'
+import ConfirmButton from './ConfirmButton'
+import Carousel from './Carousel'
+import ImageList from './ImageList'
+import { requestImages, newImages } from '~/constants/mocks'
+import { MIN_IMAGES } from './constants'
+
+const ImageVisualization = (props) => {
+  const [activeIndex, setActiveIndex] = useState(0)
+  const [submitError, setSubmitError] = useState(false)
+  const [images, setImages] = useState(requestImages.slice())
+
+  const confirmRequest = () => {}
+
+  const getMorePhotos = () => {
+    const filterDuplicates = newImages.filter(element => !images.includes(element))
+    setImages([...images, filterDuplicates[0]])
+  }
+
+  const deleteActiveIndexImage = () => {
+    const wantedImages = images.slice()
+    wantedImages.splice(activeIndex, 1)
+    setImages(wantedImages)
+
+    return wantedImages
+  }
+
+  const setSubmitErrorIfImageCountBelowMinimum = () => {
+    if (images.length < MIN_IMAGES) {
```

Comment:
This could be clearer if you do:
```javascript
return setSubmitError(images.length < MIN_IMAGES)
```

Reply:
@MPucheta please make this quick change and let me know so that I can merge the PR.